src/ProjectTemplates/BlazorTemplates.Tests/BlazorTemplates.Tests.csproj

```diff
@@ -24,23 +20,41 @@
     <TestDependsOnAspNetRuntime>true</TestDependsOnAspNetRuntime>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Include="$(SharedSourceRoot)CommandLineUtils\**\*.cs" />
+    <Compile Include="$(SharedSourceRoot)CertificateGeneration\**\*.cs" LinkBase="shared\CertificateGeneration" />
+  </ItemGroup>
+
   <ItemGroup>
     <EmbeddedResource Include="template-baselines.json" />
     <Compile Include="$(SharedSourceRoot)Process\*.cs" LinkBase="shared\Process" />
     <Compile Include="..\Shared\**" LinkBase="Helpers" />
   </ItemGroup>
 
   <ItemGroup>
+    <Content Include="playwrightSettings.*.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <DependentUpon>playwrightSettings.json</DependentUpon>
+    </Content>
+    <Content Include="playwrightSettings.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.Extensions.Configuration" />
+    <Reference Include="Microsoft.Extensions.Configuration.Json" />
```

Comment:
Why are the new references needed?

Reply:
We use json files and config for reading the configuration.

Reply:
Because we are using config to setup the different environments.

These were also put in the E2E test project props file (which I cleaned up).